style.css

```diff
@@ -1,68 +1,508 @@
+    letter-spacing: 0.5em;
+    text-transform: uppercase;
+    color: #FFFFFF;
+}
+
+.team_container{
+    width: 90%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 auto;
+}
+
+.team_img{
+    width: 50%;
+}
+
+.team_text{
+    width: 50%;
+}
+
+.team_box_title, .team_description{
+    color: rgba(255, 255, 255, 0.9);
+}
+
+.team_box{
+    width: 250px;
+    height: 250px;
+    border: 20px solid #E8C300;
+    box-sizing: border-box;
+    filter: drop-shadow(0px 0px 100px rgba(0, 0, 0, 0.5));
+    border-radius: 5px;
+    /* position: relative; */
+    margin-top: -420px;
+    margin-left: -20px;
+    margin-bottom: 212px;
+}
+
+.book{
+    width: 85%;
+    margin: 0 auto -250px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.book_text{
+    margin-top: -300px;
+}
+
+.book_us{
+    font-family: "Patrick Hand", sans-serif;
+    font-size: 36px;
+    line-height: 48px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+    border-radius: 5px;
+}
+
+.book_line{
+    margin-top: 10px;
+}
+
+.input_cintainer{
+    width: 570px;
+    height: 200px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.input{
+    width: 277px;
+    height: 60px;
+    background: #E5E5E5;
+    border-radius: 5px;
+    border: none;
+}
+
+.button_footer_book{
+    width: 563.58px;
+    height: 60px;
+    background: #E8C300;
+    border-radius: 5px;
+    border: none;
+    font-weight: bold;
+    font-size: 12px;
+    line-height: 20px;
+    color: #FFFFFF;
+    font-family: "Open Sans", sans-serif;
+}
+
+.footer_text_book{
+    font-family: "Open Sans", sans-serif;
+    font-weight: normal;
+    font-size: 12px;
+    line-height: 12px;
+    text-align: center;
+    color: rgba(51, 51, 51, 0.5);
+}
+
+.special{
+    width: 100%;
+    background: url(./images/specialtys-bg.png) center center no-repeat;
+    background-size: cover;
+    margin-top: 50px;
+}
+
+.special_container{
+    margin-top: -50px;  
+}
+
+.point{
+    width: 100%;
+    display: flex;
+    justify-content: center;
+}
+
+.po{
+    width: 10px;
+    height: 10px;
+    background: rgba(255, 255, 255, 0.5);
+    border-radius:50%;
+    margin-top: -50px;
+}
+
+.poin{
+    background: #FFFFFF;
+    margin: 0 20px;
+    margin-top: -50px;
+}
+
+.footer_img{
+    width: 100%;
+    display: flex; 
+}
+
+.fi{
+    width: 317px;
+    height: 317px;
+}
+
+.contact_description{
+    /* width: 50%; */  
+}
+
+.top_content{
+    width: 90%;
+    display: flex;
+    justify-content: space-between;
+    margin: 0 auto;
+}
+
+.contact_input{
+    width: 400px;
+}
+
+.cont_textarea{
+    width: 90%;
+    margin: 10px auto;
+}
+
+.contact_coment{
+    width: 100%;
+    height: 120px;
+    background: #E5E5E5;
+    border-radius: 5px;
+    border: none;
+}
+
+.contact_button{
+    width: 100%;
+   
+}
+
+.cont_btn{
+    width: 90%;
+    margin: 0 auto;
+}
```

Comment:
add styles for contact_content

you need to add `padding-bottom` because your button is next to the map